Server/src/models/GraphsModel.ts

```diff
@@ -28,8 +28,9 @@ export class GraphsModel {
         rawGraphData.datasetShapes = typeof rawGraphData.datasetShapes === 'string' ? JSON.parse(rawGraphData.datasetShapes) : []
         rawGraphData.datasetHiddenStatus = typeof rawGraphData.datasetHiddenStatus === 'string' ? JSON.parse(rawGraphData.datasetHiddenStatus) : []
         rawGraphData.axisVariable = typeof rawGraphData.axisVariable === 'string' ? JSON.parse(rawGraphData.axisVariable) : []
-        rawGraphData.axisMode = typeof rawGraphData.axisMode === 'string' ? JSON.parse(rawGraphData.axisMode) : []
-        rawGraphData.axisZoom = typeof rawGraphData.axisZoom === 'string' ? JSON.parse(rawGraphData.axisZoom) : []
+        rawGraphData.axisLog = typeof rawGraphData.axisLog === 'string' ? JSON.parse(rawGraphData.axisLog) : []

```

Comment:
i assume that if axisLog type is not a stiring you wanna assign default value, if its a bool you shouldnt assign an array right

Reply:
wait rawGraphData.axisLog is an array?

Reply:
All of these are arrays, or JSON, but yes this is an oversight and they should have the correct types instead of string. Admittedly, it _works_ with string, but that doesn't mean it should be there.

Reply:
This was a good catch; the type checking wasn't needed at all and has been dropped.

Reply:
All of these are strings as that's how a JSON column is returned.